packages/core/src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -229,6 +230,11 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
             this._mountComplianceSection(this._digitalRiverCheckoutData.checkoutData.sellingEntity);
 
             this._submitFormEvent = this._getDigitalRiverInitializeOptions().onSubmitForm;
+
+            const disabledPaymentMethods: string[] = [
+                features['PROJECT-4802.digital_river_paypal_support'] ? null : 'payPal'
+            ].filter((x): x is string => x !== null);
```

Comment:
This line seems equal to
```
const disabledPaymentMethods: string[] = features['PROJECT-4802.digital_river_paypal_support'] ? [] : ['payPal']
```
Can we get rid of the `filter` here?

Reply:
Hello @bc-peng. Thank you for your feedback; I changed the code to reflect your advice.